### PR TITLE
Rerender

### PR DESCRIPTION
**Pull request by dpryan79**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (`@conda-forge-admin, please rerender`)
* [ ] Ensured the license file is being packaged.

This is a tweak on #3

### CODE REVIEW

File: recipe/configure.win.patch [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+-    fi
+-    if [ ! -e src/cairo-current-win.tar.gz ]; then
+-	allok=no
+-    else
+-	echo "  unpacking current cairo"
+-	tar fxz src/cairo-current-win.tar.gz
+-	if [ ! -e src/win64/libcairo.a ]; then
+-	    allok=no
+-	fi
++        echo 'untar("src/cairo-current-win.tar.gz", exdir="./", compressed=TRUE)'|${R_HOME}/bin/R --vanilla --slave
```
Comment (bgruening) [inline recipe/configure.win.patch:16]: Ah damn, can you update this patch without the `ls` Devon.

**Comment by dpryan79** [comment]
Of course, that was for debugging:)

**Comment by dpryan79** [comment]
It's odd that `tar` didn't want to work otherwise on Windows. Anyway, this should presumably pass then.

**Comment by dpryan79** [comment]
As the saying goes, the Nth time is the charm!

**Comment by dpryan79** [comment]
@bgruening This one's good to go now.

**Comment by bgruening** [comment]
Nice, thanks a lot!

**Pull request merged**